### ISSUE TITLE
Homebrew package for Macos

### ISSUE THREAD

**Issue by Maxplosion** [prayag17/JellyPlayer#117]
Hi, 
it would be nice to have a homebrew cask package available for jellyplayer for easy installation/management.

https://github.com/Homebrew/homebrew-cask/blob/master/CONTRIBUTING.md

**Comment by prayag17** [comment]
Yeah I am planning to add this to appstores for various os once I release stable
